src/test/java/RandomGeneratorTest.java

```diff
@@ -0,0 +1,51 @@
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class RandomGeneratorTest {
+
+    private static final int minVal = 1;
+    private static final int maxVal = 99;
+    private RandomGenerator randomGenerator;
+
+    @BeforeEach
+    void setup() {
+        this.randomGenerator = new RandomGenerator();
+    }
+
+    /**
+     * Method: getNext()
+     * Verify that it always returns valid int
+     */
+    @Test
+    void testGetNext() {
+        try {
+            for (int i = minVal; i <= maxVal; i++) {
+                int val = randomGenerator.getNext();
+                assertTrue(val >= minVal);
+                assertTrue(val <= maxVal);
+            }
+        } catch (Exception e) {
+            fail();
```

Comment:
Adding a string message to the fail() method will help debug any errors. This also applies for line 45.